plugin_store/database/database.py

```diff
@@ -181,3 +181,12 @@ async def delete_plugin(self, session: "AsyncSession", id: int):
         await session.execute(delete(Version).where(Version.artifact_id == id))
         await session.execute(delete(Artifact).where(Artifact.id == id))
         return await session.commit()
+    
+    async def increment_value(self, session: "AsyncSession", name: str, isUpdate: bool):
+        if isUpdate == True:
+            query = "UPDATE artifacts SET updates = updates + 1 WHERE name == :name"
+        else:
+            query = "UPDATE artifacts SET downloads = downloads + 1 WHERE name == :name"
```

Comment:
We should avoid raw queries. We don't want to be stuck on sqlite forever.

```suggestion
        if isUpdate == True:
            await session.execute(Artifact.update().values(updates=Artifact.updates + 1).where(Artifact.name == name))
        else:
            await session.execute(Artifact.update().values(downloads=Artifact.downloads + 1).where(Artifact.name == name))
```

Reply:
Yeah that was intended to be temporary just so I could get it working quickly because the syntax was giving me trouble. Thanks!